LPS-149173 we do not want to use the tiff transformer if the content type is jpg

AliciaGarciaGarcia opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

AliciaGarciaGarcia commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-source-format/2194/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/2194/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/dd2890d9f9432ba767c55adc32da17f010758864">dd2890d9f9432ba767c55adc32da17f010758864</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/AliciaGarciaGarcia/liferay-portal/tree/LPS-149173">LPS-149173</a><br/>Branch GIT ID: <a href="https://github.com/AliciaGarciaGarcia/liferay-portal/commit/7127107ce499eacdcb407198e926a0ffd71ff7dc">7127107ce499eacdcb407198e926a0ffd71ff7dc</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-source-format/2194/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/2194/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-source-format/2194/">test-portal-source-format#2194</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-source-format/builds/2194/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-lima/liferay-portal/pull/2878">liferay-lima#2878</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1646855222">[master] ci:test:sf - AliciaGarciaGarcia &gt; liferay-lima - PR#2878 - 2022-03-25[06:53:38]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/7892/">publish-testray-report#7892</a><br/></div></div>

adolfopa commented on modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/util/TiffOrientationTransformer.java:
```diff
@@ -127,6 +134,21 @@ else if (tiffOrientationValue == _ORIENTATION_VALUE_ROTATE_270_CW) {
 
+	private static boolean _validateContentType(InputStream inputStream) {
+		if (ArrayUtil.contains(
+				_INVALID_CONTENT_TYPE,
```
Can we call it _UNSUPPORTED_MIME_TYPES? For arrays/collections we have to use the plural in the name, and the mime type is not really invalid. Also, instead of `_validate...` can we use `isContentTypeSupported` or something like that? Someone reading the code might think that there's something wrong with JPEG support.

AliciaGarciaGarcia commented:
ci:test:relevant

AliciaGarciaGarcia commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/1359/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1359/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-1.liferay.com/job/test-portal-source-format/3133/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/3133/jenkins-report.html">here</a>.</b></p>

AliciaGarciaGarcia commented:
I tried to make unitary test, but the problem I found is that I do not know how to assert that
```
Assert.assertEquals(RenderedImageUtil.readImage(_getInputStream()), TiffOrientationTransformer.transform(
			() -> _getInputStream());
```
Because RenderedImageUtil.readImage generate a different RenderedImage each time.

Some idea to test it @adolfopa ?

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9b640c3cfb997f3512a83ae79c1e24b44224f27e">9b640c3cfb997f3512a83ae79c1e24b44224f27e</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/AliciaGarciaGarcia/liferay-portal/tree/LPS-149173">LPS-149173</a><br/>Branch GIT ID: <a href="https://github.com/AliciaGarciaGarcia/liferay-portal/commit/737b601847dbde757acea44c8ecc889ce31964d5">737b601847dbde757acea44c8ecc889ce31964d5</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-1.liferay.com/job/test-portal-source-format/3133/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/3133/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-1.liferay.com/job/test-portal-source-format/3133/">test-portal-source-format#3133</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/3133/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-lima/liferay-portal/pull/2878">liferay-lima#2878</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1648415625">[master] ci:test:sf - AliciaGarciaGarcia &gt; liferay-lima - PR#2878 - 2022-03-28[05:27:39]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-1.liferay.com/job/publish-testray-report/10853/">publish-testray-report#10853</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 23 out of 26 jobs passed</h3><h3>:x: ci:test:relevant - 23 out of 27 jobs passed in 1 hour 55 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9b640c3cfb997f3512a83ae79c1e24b44224f27e">9b640c3cfb997f3512a83ae79c1e24b44224f27e</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9b640c3cfb997f3512a83ae79c1e24b44224f27e">9b640c3cfb997f3512a83ae79c1e24b44224f27e</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//3066">Acceptance Upstream DXP (master) #3066</a></p><details><summary><strong>ci:test:stable - 23 out of 26 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430370/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40611/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27983/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>23 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341053/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43507/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/343727/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43361/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44703/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430362/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/374294/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341054/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489928/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/343725/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353642/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37594/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/343724/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43509/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44705/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37592/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489929/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37595/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41042/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43359/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43508/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37591/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457172/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 23 out of 27 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/1359/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430370/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40611/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27983/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>23 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341053/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43507/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/343727/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43361/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44703/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430362/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/374294/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341054/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489928/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/343725/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353642/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37594/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/343724/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43509/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44705/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37592/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489929/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37595/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41042/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43359/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43508/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/37591/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/457172/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1359/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27983/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/27983//consoleText">AXIS_VARIABLE=0 #27983</a><pre><code>[beanshell] Created cache file in /tmp/jenkins-cached-files/-928599813.txt
[beanshell] Remote execution starting with 1 threads.
[beanshell] Executing commands: ssh -o NumberOfPasswordPrompts=0 root@test-1-34 'mkdir -p "/mnt/mfs-hdd1-10.0.10/jenkins/testray-results/production/logs/2022-03/test-1-34/test-portal-acceptance-pullrequest(master)/1359/semantic-versioning-jdk8/0/0"'
[beanshell] Average thread duration: 228ms
[beanshell] Busy slaves:0
[beanshell] Finished slaves:1
[beanshell] Target slaves:1
[beanshell] Total duration: 229
[beanshell] Remote execution completed in 229ms.
[beanshell] Remote execution starting with 1 threads.
[beanshell] Executing commands: ssh -o NumberOfPasswordPrompts=0 root@test-1-34 'mkdir -p "/mnt/mfs-hdd1-10.0.10/jenkins/testray-results/production/logs/2022-03/test-1-34/test-portal-acceptance-pullrequest(master)/1359/semantic-versioning-jdk8/0/0"'
[beanshell] Average thread duration: 226ms
[beanshell] Busy slaves:0
[beanshell] Finished slaves:1
[beanshell] Target slaves:1
[beanshell] Total duration: 226
[beanshell] Remote execution completed in 227ms.
[beanshell] Executing commands: rsync -aqz --chmod=go=rx "/opt/dev/projects/github/liferay-jenkins-ee/testray/prepared_rsync_logs"/* "test-1-34::testray-results/production/logs/"
[beanshell] Uploaded https://testray.liferay.com/reports/production/logs/2022-03/test-1-34/test-portal-acceptance-pullrequest(master)/1359/semantic-versioning-jdk8/0/0/jenkins-console.txt.gz
[beanshell] Unable to delete because file does not exist /opt/dev/projects/github/liferay-jenkins-ee/testray/prepared_s3_logs
[beanshell] Created S3 Object https://storage.cloud.google.com/testray-results/2022-03/test-1-34/test-portal-acceptance-pullrequest(master)/1359/semantic-versioning-jdk8/0/0/jenkins-console.txt.gz in 264 ms
[stopwatch] [run.current.job: 36:56.180 sec]
[beanshell] Created cache file in /tmp/jenkins-cached-files/-928599813.txt

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:13846: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build-test-batch.xml:7274: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build-test-batch.xml:7355: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-portal/build-test-batch.xml:946: Unable to prepare test build.</code></pre></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 9b640c3cfb997f3512a83ae79c1e24b44224f27e:</strong></summary><ol><li><div><h5><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430370/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/430370//consoleText">AXIS_VARIABLE=0 #430370</a><div><p>Startup error: <strong>Unresolved Requirement(s)</strong></p><pre><code> [parallel]     {name: kaleoDefinitionVersionLocalService, target: null}
 [parallel]     {name: unproxiedWorkflowDefinitionManager, target: (proxy.bean=false)}
 [parallel]   }
 [parallel]   Declarative Service {id: 5108, name: com.liferay.portal.workflow.kaleo.designer.web.internal.permission.KaleoDesignerPermission, unsatisfied references: 
 [parallel]     {name: PortletResourcePermission, target: (resource.name=com.liferay.portal.workflow.kaleo.designer)}
 [parallel]   }
 [parallel]   Declarative Service {id: 5104, name: com.liferay.portal.workflow.kaleo.designer.web.internal.application.list.KaleoDesignerPanelApp, unsatisfied references: 
 [parallel]     {name: Portlet, target: (javax.portlet.name=com_liferay_portal_workflow_kaleo_designer_web_portlet_KaleoDesignerPortlet)}
 [parallel]   }
 [parallel]   Declarative Service {id: 5124, name: com.liferay.portal.workflow.kaleo.designer.web.internal.verify.KaleoDesignerWebVerifyProcess, unsatisfied references: 
 [parallel]     {name: _kaleoDefinitionVersionLocalService, target: null}
 [parallel]     {name: _release, target: (&amp;(release.bundle.symbolic.name=com.liferay.portal.workflow.kaleo.designer.web)(release.schema.version=1.0.2))}
 [parallel]   }
 [parallel] 2022-03-28 13:57:38.787 INFO  [main][ServiceProxyFactory:68] Running "Spring Extender Unavailable Component Checker" check result: No issues were found.
 [parallel] 2022-03-28 14:02:38.788 ERROR [main][ServiceProxyFactory:278] Service "com.liferay.portal.kernel.util.MimeTypes" is unavailable in 300000 milliseconds while setting field "_mimeTypes" for class "com.liferay.portal.kernel.util.MimeTypesUtil", will retry...
 [parallel] 2022-03-28 14:07:38.788 ERROR [main][ServiceProxyFactory:278] Service "com.liferay.portal.kernel.util.MimeTypes" is unavailable in 300000 milliseconds while setting field "_mimeTypes" for class "com.liferay.portal.kernel.util.MimeTypesUtil", will retry...
 [parallel] 2022-03-28 14:12:38.789 ERROR [main][ServiceProxyFactory:278] Service "com.liferay.portal.kernel.util.MimeTypes" is unavailable in 300000 milliseconds while setting field "_mimeTypes" for class "com.liferay.portal.kernel.util.MimeTypesUtil", will retry...
 [parallel] 2022-03-28 14:17:38.789 ERROR [main][ServiceProxyFactory:278] Service "com.liferay.portal.kernel.util.MimeTypes" is unavailable in 300000 milliseconds while setting field "_mimeTypes" for class "com.liferay.portal.kernel.util.MimeTypesUtil", will retry...</code></pre></div></div></li></ol></div></li><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/40611/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>626 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/40611//consoleText">AXIS_VARIABLE=0 #40611</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/40611//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Unable to find default image scaler
  at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
  at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
  at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
  at java.nio.file.Files.walkFileTree(Files.java:2670)
  at java.nio.file.Files.walkFileTree(Files.java:2742)
  at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/1359/">test-portal-acceptance-pullrequest(master)#1359</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1359/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-lima/liferay-portal/pull/2878">liferay-lima#2878</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1648463146">[master] ci:test:relevant - AliciaGarciaGarcia &gt; liferay-lima - PR#2878 - 2022-03-28[05:27:23]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/3870/">publish-testray-report#3870</a><br/></div></div>

AliciaGarciaGarcia commented:
ci:test:relevant

AliciaGarciaGarcia commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/1407/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1407/jenkins-report.html">here</a>.</b></p>

AliciaGarciaGarcia commented:
@adolfopa I changed the names and some other issues about the inputStream
And added a test for both behaviors. In the end I do not used the tiff image because was giving me headaches, and used instead a png with exif values.

Tell me your thoughts about it.
Thanks!

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-12.liferay.com/job/test-portal-source-format/4214/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/4214/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-12.liferay.com/job/test-portal-source-format/4214/">test-portal-source-format#4214</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/4214/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-lima/liferay-portal/pull/2878">liferay-lima#2878</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1651065459">[master] ci:test:sf - AliciaGarciaGarcia &gt; liferay-lima - PR#2878 - 2022-03-30[03:10:11]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-12.liferay.com/job/publish-testray-report/2212/">publish-testray-report#2212</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 24 out of 25 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 25 out of 26 jobs passed in 1 hour 33 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2f6c3e638afbe31d63af5c62dfe5ed43150cb367">2f6c3e638afbe31d63af5c62dfe5ed43150cb367</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2f6c3e638afbe31d63af5c62dfe5ed43150cb367">2f6c3e638afbe31d63af5c62dfe5ed43150cb367</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//3072">Acceptance Upstream DXP (master) #3072</a></p><details><summary><strong>ci:test:stable - 24 out of 25 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28723/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28721/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28719/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/485786/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403637/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403635/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453396/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479222/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41489/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44108/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353869/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44385/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31438/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341312/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403636/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453395/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479221/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341313/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403638/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/485785/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/485784/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44107/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28720/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44384/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28722/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 24 out of 26 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/1407/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28723/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28721/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28719/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/485786/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403637/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403635/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453396/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479222/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41489/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44108/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353869/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44385/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31438/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341312/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403636/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/453395/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479221/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341313/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/403638/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/485785/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/485784/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44107/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28720/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44384/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28722/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1407/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 2f6c3e638afbe31d63af5c62dfe5ed43150cb367:</strong></summary><ol><li><div><h5><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28723/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>626 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/28723//consoleText">AXIS_VARIABLE=0 #28723</a><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/28723//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Unable to find default image scaler
  at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
  at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
  at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
  at java.nio.file.Files.walkFileTree(Files.java:2670)
  at java.nio.file.Files.walkFileTree(Files.java:2742)
  at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/1407/">test-portal-acceptance-pullrequest(master)#1407</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1407/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-lima/liferay-portal/pull/2878">liferay-lima#2878</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1651090454">[master] ci:test:relevant - AliciaGarciaGarcia &gt; liferay-lima - PR#2878 - 2022-03-30[03:10:04]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/3966/">publish-testray-report#3966</a><br/></div></div>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-35.liferay.com/job/forward-pullrequest/128//consoleFull)
